.github/workflows/PR-build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           echo "MDS=$(git diff --name-only --diff-filter=ACMRTUXB $(git merge-base origin/${{ github.event.pull_request.base.ref }} ${{ github.event.pull_request.head.sha }}) ${{ github.event.pull_request.head.sha }} | grep .md$ | grep -v vendor/ | xargs)" >> $GITHUB_ENV
 
       - name: Install markdown-link-check
-        run: npm install -g markdown-link-check
+        run: npm install -g markdown-link-check@3.12.2
```

Comment:
when do we expect to be able to unpin this?

Reply:
We shall unpin once the issue is resolved upstream - https://github.com/tcort/markdown-link-check/issues/369